### ISSUE TITLE
Some bees not producing combs

### ISSUE THREAD

**Issue by NiSiSuinegEht** [Team-Resourceful/ResourcefulBees#206]
Using Delivery Inc. v1.0.14 and Resourceful Bees v0.6.5b

I'm finding certain bees won't produce combs, even if they're the only bee in the apiary or hive.

Bees that I've found to not produce combs so far:

Zombie bee

There may be more, since I've not found all of them. If these bees have special, undocumented conditions for producing combs, I'm not aware of them.

[XPosted on Delivery Inc issue tracker.](https://github.com/henkelmax/delivery-inc/issues/6)

**Comment by Vaerys-Dawn** [comment]
You will need to give more information than that, make sure the bees have the correct flowers available to them. Do they have combs that they can produce? some bees don't have combs.

There are no requirements for producing combs other than having a hive and a valid flower

**Comment by NiSiSuinegEht** [comment]
According to Zombie.json:

`{
  "flower": "tag:minecraft:bee_growables",
  "maxTimeInHive": 600,
  "hasHoneycomb": true,
  "baseLayerTexture": "zombie/zombie_bee",
  "traits": ["zombie"],
  "ColorData": {
    "isBeeColored": false,
    "honeycombColor": "#2F4E32"
  },
  "MutationData": {
    "hasMutation": true,
    "mutationInput": "minecraft:stone",
    "mutationOutput": "minecraft:cobblestone",
    "mutationType": "BLOCK_TO_BLOCK"
  },
  "CentrifugeData": {
    "hasCentrifugeOutput": true,
    "mainOutput": "minecraft:rotten_flesh"
  },
  "SpawnData": {
    "canSpawnInWorld": true,
    "biomeWhitelist": "tag:OVERWORLD",
    "biomeBlacklist": "tag:ocean",
    "lightLevel": "NIGHT"
  },
  "BreedData": {
    "isBreedable": true
  },
  "TraitData": {
    "hasTraits": true
  }
}`

I've tried a variety of flowers to no avail.  The don't even increase the honey level of a hive when leaving it.  They certainly seem to be pollenating flowers, judging by their animations.

**Comment by Vaerys-Dawn** [comment]
they need things like wheat and carrots to pollinate, are you using those?

**Comment by NiSiSuinegEht** [comment]
> they need things like wheat and carrots to pollinate, are you using those?

None of the other resource bees I have need those, is there something that would indicate this particular bee does?

I have a second apiary with diamond, emerald, iron, copper, osmium, redstone, and coal bees happily producing combs from just poppies and dandelions.

I just tested in an installation with just Resource Bees, and you are right, it actually needs wheat or carrots to produce.  Why is this different from every other bee I've found?  The Beepeida doesn't mention anything about that, there's nothing in the JEI data to suggest it either. 

Regardless, I'll close the issue now.  Thanks for the information.
